compiler/src/iree/compiler/Codegen/Common/GPU/ExpandGPUOps.cpp

```diff
@@ -22,6 +24,7 @@ namespace mlir::iree_compiler {
 namespace {
 
 struct ExpandGPUOpsPass final : impl::ExpandGPUOpsPassBase<ExpandGPUOpsPass> {
+  // Apply AMD GPU targetting patterns
```

Comment:
nit: typos + missing comma (https://llvm.org/docs/CodingStandards.html#commenting)
```suggestion
  // Apply AMD GPU targeting patterns.
```